packages/rpc-graphql/src/schema/instruction.ts

```diff
@@ -553,6 +553,18 @@ export const instructionTypeDefs = /* GraphQL */ `
         maximumFee: Int
     }
 
+    """
+    SplToken-2022: WithdrawWithheldTokensFromMint instruction
+    """
+    type SplTokenWithdrawWithheldTokensFromMint implements TransactionInstruction {
+        programId: Address
+        feeRecipient: Account
+        mint: Account
+        withdrawWithheldAuthority: Account
+        multisigWithdrawWithheldAuthority: Account
+        signers: [Address]
```

Comment:
Just calling out again here we're going to want to do `[Account]` for signers. I think for now just merge with `[Address]` and when I finish the `resolveMultipleAccounts` PR I'll go back and update these.

This way you're not blocked. What do you think?

Reply:
Sounds good. I'll update the test and get that merged.